DeviceManager/DeviceManager/View/LoginVC.swift

```diff
@@ -8,29 +8,68 @@
 
 import UIKit
 
-let lll = ["email" , "Pass"]
-class LoginVC:BaseVC,UITableViewDelegate,UITableViewDataSource{
-@IBOutlet weak var tableView: UITableView!
+let labelField = LoginVM()
```

Comment:
naming convention: If you are declaring a variable of VM instance, let the variable be of same name like
`let loginVM = LoginVM()`